resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/internal/CompletionStageInvoke.java

```diff
@@ -1,176 +0,0 @@
-package org.jboss.resteasy.client.jaxrs.internal;
-
-import org.jboss.resteasy.spi.NotImplementedYetException;
-
-import javax.ws.rs.HttpMethod;
-import javax.ws.rs.client.Entity;
-import javax.ws.rs.core.GenericType;
-import javax.ws.rs.core.Response;
-import java.util.concurrent.CompletionStage;
-
-/**
- * An invoker for asynchronous invocation of HTTP methods that returns {@link CompletionStage}
- *
- * @author <a href="mailto:anilgursel@gmail.com">Anil Gursel</a>
- * @version $Revision: 1 $
- */
-public class CompletionStageInvoke
```

Comment:
The reason I created this class was just to align with current design of RestEasy where it has `AsynchronousInvoke`. But, I see no issue getting rid of it.  It is cleaner not to have it.